core/iwasm/aot/aot_loader.c

```diff
@@ -123,6 +123,42 @@ GET_U64_FROM_ADDR(uint32 *addr)
     return u.val;
 }
 
+#if defined(BUILD_TARGET_XTENSA) && (WASM_ENABLE_AOT_LOAD_FROM_VRAM != 0)
```

Comment:
It seems that the feature is only related to VRAM, it doesn't matter with XTENSA, some other targets/platforms may also need the feature, my understanding is that here (and many other places) we can just use WASM_ENABLE_AOT_LOAD_FROM_VRAM to control the code:
```C
#if WASM_ENABLE_AOT_LOAD_FROM_VRAM != 0
...
#endif
```